pkg/backend/apply.go

```diff
@@ -125,6 +125,16 @@ func PreviewThenPrompt(ctx context.Context, kind apitype.UpdateKind, stack Stack
 		return plan, changes, nil
 	}
 
+	if kind == apitype.UpdateUpdate &&
+		countResources(events, op.Opts.Display) == 0 {
+
+		emptyStackWarningMessage := "\b" + op.Opts.Display.Color.Colorize(
+			fmt.Sprintf("%sinfo:%s it appears that there are no resources in this update\n",
```

Comment:
```suggestion
			fmt.Sprintf("%sinfo:%s there are no resources in this update\n",
```

is _resources_ the right word here? or should it be _resource changes_?

Reply:
The issue that was opened only mentioned deploying  a stack that only contains a stack resource and nothing else.

> can we detect when someone is running an update with nothing being deployed beyond the stack itself and warn the user about this?

I may have misunderstood your ask.

Reply:
@dixler ah yah, thx for reminding me! this is confusing! i tried an edit, what do you think?

Reply:
imo users don't specify the stack resource so it's an internal detail that can be neglected